pkg/tagger/replay/tagger.go

```diff
@@ -0,0 +1,174 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package replay
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/api/response"
+	pb "github.com/DataDog/datadog-agent/pkg/proto/pbgo"
+	pbutils "github.com/DataDog/datadog-agent/pkg/proto/utils"
+	"github.com/DataDog/datadog-agent/pkg/status/health"
+	"github.com/DataDog/datadog-agent/pkg/tagger/collectors"
+	"github.com/DataDog/datadog-agent/pkg/tagger/telemetry"
+	"github.com/DataDog/datadog-agent/pkg/tagger/types"
+	"github.com/DataDog/datadog-agent/pkg/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// Tagger stores tags to entity as stored in a replay state.
+type Tagger struct {
+	store *tagStore
+
+	ctx    context.Context
+	cancel context.CancelFunc
+
+	health          *health.Handle
+	telemetryTicker *time.Ticker
+}
+
+// NewTagger returns an allocated tagger. You still have to run Init()
+// once the config package is ready.
+func NewTagger() *Tagger {
+	return &Tagger{
+		store: newTagStore(),
+	}
+}
+
+// Init initializes the connection to the replay tagger and starts watching for
+// events.
+func (t *Tagger) Init() error {
+	t.health = health.RegisterLiveness("tagger")
+	t.telemetryTicker = time.NewTicker(1 * time.Minute)
+
+	t.ctx, t.cancel = context.WithCancel(context.Background())
+
+	// I do not think there's a need for any async work
+	// go t.run()
+
+	return nil
```

Comment:
i think an `error` will never be returned, maybe can rm

Reply:
We have to satisfy the interface, so we'll have to return `nil`.